### DISCUSSION TITLE
量子電腦的原理、挑戰與未來衝擊

### DISCUSSION THREAD

**Discussion by hill0106** [1101-GS3073-C/Python-Community-GS3073-C#473] [General]
近年來量子電腦的發展越來越盛行，因為量子電腦屬於較新的領域，但是對未來的幫助是非常有影響力的！在這邊和大家分享一下有關量子電腦的簡單介紹～

https://www.narlabs.org.tw/xcscience/cont?xsmsid=0I148638629329404252&sid=0M103505917643100370&sq=%E9%87%8F%E5%AD%90%E9%9B%BB%E8%85%A6

**Comment by williamyen0623** [comment]
希望以後使用電腦不會陷入量子糾纏

**Comment by manipulateed** [comment]
網址可以直接複製貼上就好了歐(別人教我的)，不然點進去連不到。

**Reply by hill0106** [reply]
感謝～已經更改了😀

**Comment by brian431** [comment]
> 近年來量子電腦的發展越來越盛行，因為量子電腦屬於較新的領域，但是對未來的幫助是非常有影響力的！在這邊和大家分享一下有關量子電腦的簡單介紹～
> 
> 
> 
> https://www.narlabs.org.tw/xcscience/cont?xsmsid=0I148638629329404252&sid=0M103505917643100370&sq=%E9%87%8F%E5%AD%90%E9%9B%BB%E8%85%A6

量子電腦的研究有非常多的瓶頸包括增加位元數、維持穩定量子態等。可能要再出現一個像夏儂那樣的天才才能使這項技術有近一步的突破。

**Reply by hill0106** [reply]
雖然還有很多要突破的地方，但是現在全世界都投入這麼多的研究，還是很期待未來的發展🤩

**Comment by boson13579** [comment]
這裡有大佬能解釋為什麼量子運算的次數是總組合數的根號次嗎？我看完了文章還是沒有很懂

**Reply by brian431** [reply]
大概的想法是因為量子可以同時表示0跟1，使得需要運算的次數大量減少。

**Comment by lx3211** [comment]
量子電腦目前只能在絕對零度的低溫環境才能運作，看來離普及之日還有好久

**Comment by Chenszuhong** [comment]
這感覺是一個很有趣的議題，值得學習

**Comment by BingRong1217** [comment]
在未來量子電腦是否會成為趨勢
